docs/ref/contrib/admin/index.txt

```diff
@@ -1305,6 +1305,16 @@ subclass::
         WHERE (first_name ILIKE '%john%' OR last_name ILIKE '%john%')
         AND (first_name ILIKE '%lennon%' OR last_name ILIKE '%lennon%')
 
+    If you don't want to split your search query into words, you can surround
```

Comment:
Please add `..versionadded:: 3.2` . (notes in docs [here](https://docs.djangoproject.com/en/dev/internals/contributing/writing-documentation/#documenting-new-features))

Also as this is labeled as a new feature please also add to the release notes (3.2.txt).

Reply:
Thanks for your feedback ! It's added.